conference.js

```diff
@@ -2361,9 +2361,7 @@ export default {
      * NOTE: Should be used after conference.init
      */
     logEvent(name, value, label) {
-        if (JitsiMeetJS.analytics) {
-            JitsiMeetJS.analytics.sendEvent(name, {value, label});
-        }
+        sendEvent(name, {value, label});
```

Comment:
We initialize sendEvent as "export const sendEvent = analytics.sendEvent.bind(analytics);". Are you sure there's no way that the analytics object is null or undefined at this point? Should we add a check in the initialization?

Reply:
I think after https://github.com/jitsi/lib-jitsi-meet/pull/605 is merged, the analytics object should always exists. I don't think there should be such an use case, but maybe I'm missing something. Do you have anything in mind?